config/vault.go

```diff
@@ -83,6 +91,34 @@ type VaultConfig struct {
 	// refresh dynamic, non-renewable leases, measured as a fraction of the lease
 	// duration.
 	LeaseRenewalThreshold *float64 `mapstructure:"lease_renewal_threshold"`
+
+	// If Token is empty and K8SAuthRoleName is set, it means to use
+	// k8s vault auth method.
+	//
+	// The kubernetes auth method can be used to authenticate with Vault
+	// using a Kubernetes Service Account Token. This method of
+	// authentication makes it easy to introduce a Vault token into
+	// a Kubernetes Pod.
+	//
+	// This can also be set via the VAULT_K8S_AUTH_ROLE_NAME.
+	K8SAuthRoleName *string `mapstructure:"k8s_auth_role_name"`
+	// K8SServiceAccountTokenPath is the path of file that contains
+	// a K8SServiceAccountToken. It will be ignored if K8SServiceAccountToken
+	// is set.
+	//
+	// Default value is "/var/run/secrets/kubernetes.io/serviceaccount/token".
```

Comment:
Should be /run/secrets/kubernetes.io/serviceaccount/token

Reply:
@yellowmegaman changed